kolibri/plugins/coach/assets/src/views/exams-page/preview-exam-modal.vue

```diff
@@ -4,7 +4,7 @@
     <div class="question-selector-container">
       <div class="question-selector">
         <div>
-          {{ $tr('numQuestions', { num: exam.question_count })}}<icon-button :text="$tr('randomize')" @click="$emit('randomize')" :primary="true"/>
```

Comment:
The randomize feature only exists within the exam preview during exam creation. https://projects.invisionapp.com/share/7D8ZDY4EN#/screens/222425512